common/src/main/java/com/wynntils/mc/event/InventoryRenderEvent.java

```diff
@@ -5,17 +5,20 @@
 package com.wynntils.mc.event;
 
 import com.mojang.blaze3d.vertex.PoseStack;
-import com.wynntils.framework.events.Event;
 import net.minecraft.client.gui.screens.Screen;
 import net.minecraft.world.inventory.Slot;
+import net.minecraftforge.eventbus.api.Event;
 
 public class InventoryRenderEvent extends Event {
-    private final Screen screen;
-    private final PoseStack poseStack;
-    private final int mouseX;
-    private final int mouseY;
-    private final float partialTicks;
-    private final Slot hoveredSlot;
+    private Screen screen;
+    private PoseStack poseStack;
+    private int mouseX;
+    private int mouseY;
+    private float partialTicks;
+    private Slot hoveredSlot;
+
+    public InventoryRenderEvent() {
+    }
 
```

Comment:
This change is fundamentally broken, and can hopefully be reverted if I can get the constructor transformer thingie to work on fabric.